workflows/multiomics/integration/initialize_integration/config.vsh.yaml

```diff
@@ -46,6 +46,9 @@ functionality:
           type: string
           required: false
           description: Column name in .var matrix that will be used to select which genes to run the PCA on.
+        - name: "--pca_overwrite"
+          type: boolean_true
+          description: "Allow overwriting slots for PCA output."
```

Comment:
Do you want the default value for this to be true?

```yaml
        - name: "--pca_overwrite"
          type: boolean
          description: "If the PCA output already exists (in .obsm, .varm and .uns), allow overwriting the values in those slots."
          default: true
```

Reply:
My first thought was yes, because the main purpose of this workflow is to re-analyse data that has been processes by the full pipeline already. But I get your point. 

On an unrelated note: I think we should/could add an `--overwrite` argument to many other components...

Reply:
So your first thought was yes but the conclusion was no?

Reply:
Ok, so my conclusion is to set the default to `true` for the `multisample` pipeline, but for the integration pipeline I would not add the default `true`